### PR TITLE
Update proposal  based on the doc. Update star count

### PR DESCRIPTION
**Pull request by antweiss**
#### Description

This PR fixes #    

#### Checklist

- [] [DCO](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) signed.
- [] Title of the PR starts with area name, e.g. `meshery:`, `mesheryctl:`, `ui:`.

### CODE REVIEW

File: proposals/meshery.md [hunk 1]
```diff
@@ -1,85 +1,129 @@
...
+-   Website:  [https://meshery.io](https://meshery.io/)
+
+-   Release Methodology and Mechanics: Documented at <https://meshery.io/releases>
+
+Social Media Accounts
+---------------------
+
+-   Twitter:  <https://twitter.com/mesheryio>
+
+-   Google Groups:  <https://groups.google.com/forum/#!forum/projectcontour-announce>
```
Comment (kushthedude) [inline proposals/meshery.md:92]: I don't think it is for meshery ?

File: proposals/meshery.md [hunk 2]
```diff
@@ -1,85 +1,129 @@
...
+Project Timeline and Snapshot
+-----------------------------
+
+-   Layer5 launched and open sourced Meshery in January 2019 as a multi-service mesh manager
 
-== In-Flight Features
+Users
+-----
+
+-   Ziglu, TicketMaster, HPE
```
Comment (kushthedude) [inline proposals/meshery.md:42]: Can we add HashiCorp also over here ?

File: proposals/meshery.md [hunk 3]
```diff
@@ -1,85 +1,129 @@
-== Meshery Proposal
+Meshery CNCF Proposal
 
-*Name of project:* Meshery
+Name of project: Meshery
 
-*Description:* Meshery is an open source, multi-service mesh management plane.
+Description: Meshery is an open source multi-service mesh management plane.
```
Comment (kushthedude) [inline proposals/meshery.md:5]: I think they should be bold.

File: proposals/meshery.md [hunk 4]
```diff
@@ -1,85 +1,129 @@
...
 
-=== Alignment with CNCF - Why does CNCF need a multi-service mesh management plane?
+Alignment with CNCF - Why does CNCF need a multi-service mesh management plane?
+===============================================================================
```
Comment (kushthedude) [inline proposals/meshery.md:8]: Can we remove this line, it's making the above line too huge?
Screenshot 2020-04-06 at 1 47 00 AM

**Comment by antweiss** [comment]
@kushthedude I made the proposed improvements. thanks for looking at this

File: proposals/meshery.md [hunk 5]
```diff
@@ -1,85 +1,126 @@
...
+-   Ability to run performance tests on applications
+
+-   Persist and compare results of different performance tests
+
+CNCF Donation Details
+---------------------
+
+-   Preferred Maturity Level: Sandbox
+
+-   Sponsors: @leecalcote, @justincormack, @michellenoorali,
```
Comment (kushthedude) [inline proposals/meshery.md:74]: Can we add the links to their GitHub/Social Profiles ?

Comment (leecalcote) [reply proposals/meshery.md:74]: Sounds good. FYI - convention in the CNCF is to use links to Twitter handles.

Comment (antweiss) [reply proposals/meshery.md]: Added Twitter handles with links

**Comment by antweiss** [comment]
Anything else holding this from being approved?

**Comment by kushthedude** [comment]
> Anything else holding this from being approved?

Hi @antweiss, I personally feel like to hold it for a day or two more. As we are already at 288 stars and it won't be much amount of time when we will be at 300 stars 🤩 . We can directly update the proposal from your PR & prepare to get into CNCF's sandbox.

@leecalcote What do you suggest?

**Comment by antweiss** [comment]
288! :)) when did that happen!? Sounds good to me.

**Comment by leecalcote** [comment]
First of all, I'll suggest that 🤩 I'm seeing nothing but stars. Way to be, @kushthedude! It takes a team here for sure.

And then, yes, I think we can merge here really at any point. There are a few notable items to clear out of the way before submitting this project proposal. We'll undoubtedly have to come back to update stats and make a few statements more boldly in the proposal.

Thanks for your help here. Looking good.

**Comment by leecalcote** [comment]
Gents, let’s go ahead and bring these changes in. No doubt we’ll have others in the future. Thanks for advancing this, @antweiss 👍

**Pull request merged**